cvat/apps/engine/permissions.py

```diff
@@ -288,15 +298,19 @@ def get_scopes(request: ExtendedRequest, view: ViewSet, obj: Project | None):
             ('dataset', 'POST'): Scopes.IMPORT_DATASET,
             ('append_dataset_chunk', 'HEAD'): Scopes.IMPORT_DATASET,
             ('append_dataset_chunk', 'PATCH'): Scopes.IMPORT_DATASET,
-            ('annotations', 'GET'): Scopes.EXPORT_ANNOTATIONS,
-            ('dataset', 'GET'): Scopes.IMPORT_DATASET if request.query_params.get('action') == 'import_status' else Scopes.EXPORT_DATASET,
-            ('export_dataset_v2', 'POST'): Scopes.EXPORT_DATASET if is_dataset_export(request) else Scopes.EXPORT_ANNOTATIONS,
-            ('export_backup', 'GET'): Scopes.EXPORT_BACKUP,
-            ('export_backup_v2', 'POST'): Scopes.EXPORT_BACKUP,
+            ('initiate_dataset_export', 'POST'): Scopes.EXPORT_DATASET if is_dataset_export(request) else Scopes.EXPORT_ANNOTATIONS,
+            ('initiate_backup_export', 'POST'): Scopes.EXPORT_BACKUP,
             ('import_backup', 'POST'): Scopes.IMPORT_BACKUP,
             ('append_backup_chunk', 'PATCH'): Scopes.IMPORT_BACKUP,
             ('append_backup_chunk', 'HEAD'): Scopes.IMPORT_BACKUP,
             ('preview', 'GET'): Scopes.VIEW,
+            ('download_dataset', 'GET'): Scopes.DOWNLOAD_EXPORTED_FILE,
+            ('download_backup', 'GET'): Scopes.DOWNLOAD_EXPORTED_FILE,
+            # FUTURE-TODO: delete this after dropping support for deprecated API
+            ('annotations', 'GET'): Scopes.EXPORT_ANNOTATIONS,
```

Comment:
`GET annotations` is not supposed to be deleted, though, is it? It's still used for fetching raw annotations.

Reply:
These are project permissions, there is no API to retrieve project annotations.

Reply:
Ah, my bad.